keras/src/backend/openvino/core.py

```diff
@@ -22,6 +22,8 @@
 SUPPORTS_RAGGED_TENSORS = False
 IS_THREAD_SAFE = True
 
+_print = print
```

Comment:
nitpick: instead of doing this, let's use `__builtins__`:

```python
def print(*args, **kwargs):
  return __builtins__.print(*args, **kwargs)
```